src/executor/mod.rs

```diff
@@ -114,20 +120,89 @@ pub type BoxedExecutor = BoxStream<'static, Result<DataChunk, ExecutorError>>;
 /// The builder of executor.
 #[derive(Clone)]
 pub struct ExecutorBuilder {
+    context: Arc<Context>,
     storage: StorageImpl,
 }
 
 impl ExecutorBuilder {
     /// Create a new executor builder.
     pub fn new(storage: StorageImpl) -> ExecutorBuilder {
```

Comment:
Do we even need the method now?

Reply:
Not really, just for backward compatibility. Remove it should be nicer.